src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+
+.calendar {
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $paddings;
+  max-width: $calendar-width;
+  perspective: $cell-size;
+
+  &__day {
+    box-sizing: border-box;
+    @include cell($cell-size);
+    border: 1px solid black;
+    background-color: #eee;
+    text-align: center;
+    line-height: $cell-size;
+    transition: $transition-time;
+
+    &:hover {
+      background-color: $c-pink;
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+
+    @for $number from 1 through 31 {
+      &:nth-child(#{$number})::before {
+        content: "#{$number}";
+      }
+    }
+
+    @each $day, $day-index in $days {
+      .calendar--start-day--#{$day} &:first-child {
```

Comment:
Use "&" here, like &--start-day--#{$day}

Reply:
got it!